Trying to build from source but getting TypeError

### Bootstraptable version(s) affected

develop 2125004

### Description

So I cloned repository, run `npm run build` and it failed with `sh: 1: run-s: not found`

First thing that I did is I searched here for any issues, found one that suggested to do `yarn add npm-run-all --dev`

After I did that I am getting following error wall:
```
Oops! Something went wrong! :(

ESLint: 8.21.0

TypeError: Module.createRequire is not a function
    at Object.<anonymous> (/home/max/p/bootstrap-table/node_modules/@eslint/eslintrc/dist/eslintrc.cjs:2383:26)
    at Module._compile (/home/max/p/bootstrap-table/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (/home/max/p/bootstrap-table/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
    at Object.<anonymous> (/home/max/p/bootstrap-table/node_modules/eslint/lib/cli-engine/cli-engine.js:33:5)
    at Module._compile (/home/max/p/bootstrap-table/node_modules/v8-compile-cache/v8-compile-cache.js:192:30)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! bootstrap-table@1.20.2 lint:js: `eslint src`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the bootstrap-table@1.20.2 lint:js script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/max/.npm/_logs/2022-08-02T22_28_16_563Z-debug.log
ERROR: "lint:js" exited with 2.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bootstrap-table@1.20.2 lint: `run-s lint:*`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the bootstrap-table@1.20.2 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/max/.npm/_logs/2022-08-02T22_28_16_593Z-debug.log
ERROR: "lint" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bootstrap-table@1.20.2 build: `run-s lint clean *:build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the bootstrap-table@1.20.2 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/max/.npm/_logs/2022-08-02T22_28_16_630Z-debug.log
```

I really want filters to be fixed but don't want to wait for next release...

### Example(s)

_No response_

### Possible Solutions

_No response_

### Additional Context

_No response_

--- a/README.md
+++ b/README.md
@@ -114,9 +114,18 @@ And then open: http://localhost:8081/bootstrap-table-examples
 
 ## Local build
 
+Be sure to use a current version of yarn/npm.
 To build bootstrap-table locally please run:
 
+### Yarn
 ```
+yarn install
+yarn build
+```
+
+### Npm
+```
+npm install
 npm run build
 ```
 

